src/components/Input/__snapshots__/Input.spec.js.snap

```diff
@@ -240,6 +247,7 @@ exports[`Input should render with inline styles when passed the inline prop 1`]
   <input
     className="circuit-0 circuit-1"
     disabled={false}
+    textAlign="left"
```

Comment:
I think you need to add this list to the blacklist of the HTML element so it doesn't get passed down to the input itself. You should find this on the `HTMLElement`.

Reply:
Yes, look in the input component for this section:

```javascript
     <InputElement
       {...{ ...props, disabled }}
       blacklist={{ optional: true, invalid: true }}
     />
```

Extend the blacklist to include your prop please :)